src/Illuminate/Cache/RateLimiting/Limit.php

```diff
@@ -104,7 +106,7 @@ public static function perHour($maxAttempts, $decayHours = 1)
      */
     public static function perDay($maxAttempts, $decayDays = 1)
     {
-        return new static('', $maxAttempts, 60 * 60 * 24 * $decayDays);
+        return new static('', $maxAttempts, SECONDS_PER_MINUTE * MINUTES_PER_HOUR * HOURS_PER_DAY * $decayDays);
```

Comment:
Maybe we should also add shorthands for `SECONDS_PER_DAY` and `SECONDS_PER_HOUR`? These values are quite  common.

Reply:
Not in Laravel itself, I thought it would be useful for userland code.

Reply:
I agree, though, I was hesitant to add it right away in case that would make rejection more likely.

Reply:
Isn't this exactly the file and line where `SECONDS_PER_DAY` would get used? :) Although for me values like 3600, 86400 or 24*60*60 are easier to recognize than any text alias.

Reply:
@tontonsb Sometimes one's just unable to see the wood for the trees 🤦🏻😅